api/v1/logstorage_types.go

```diff
@@ -140,6 +140,12 @@ type Retention struct {
 	// Default: 91
 	// +optional
 	ComplianceReports *int32 `json:"complianceReports"`
+
+	// DNSLogs configures the retention period for DNS logs, in days.  Logs written on a day that started at least this long ago
+	// are removed.  To keep logs for at least x days, use a retention period of x+1.
+	// Default: 8
+	// +optional
+	DNSLogs *int32 `json:"dnsLogs"`
```

Comment:
Flow logs are called "Flows"
Dns logs are called "DNSLogs"

Maybe we want to rename it to simply "DNS"?
@caseydavenport

Reply:
It's a bit of a bummer that they're not consistently named but I think "DNSLogs" is probably a better name for this than "DNS", which is quite vague and less obvious what it might do.